hr_expense_invoice/models/hr_expense.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class HrExpense(models.Model):
+    _inherit = 'hr.expense'
+
+    invoice_id = fields.Many2one(
+        comodel_name="account.invoice",
+        string='Invoice',
```

Comment:
The string param is not necessary on v12 (in this case)

Reply:
Sorry, but why? invoice_id is a new field for hr.expense also, why no need `string='Invoice`

Reply:
By the way, I also see `oldname='invoice'`. Do we still need it for migration to v12?
What is it actually used for?

Reply:
because the string is computed using the field name, so there is any need to explicitly set the string param

Reply:
@hugho-ad, you mean "invoice_id" will be display as string "Invoice" anyway, right?

Reply:
Yes @kittiu

Reply:
Note. But in this case, may I also change it to "Vendor Bill", and it is domain to in_invoice + open status. I think Vendor Bill is also more meaningful. :)

Reply:
Yes @kittiu

When the label is different, the string parameter is necessary.